app/helpers/public_body_helper.rb

```diff
@@ -18,6 +18,10 @@ def public_body_not_requestable_reasons(public_body)
       reasons.push _('Freedom of Information law does not apply to this authority, so you cannot make a request to it.')
     end
 
+    if public_body.not_requestable?
+      reasons.push _('We are unable to make requests to this authority.')
```

Comment:
Not 100% on the phrasing here but don't have an immediate suggestion. @HelenWDTK any thoughts?